infrastructure/controller/pkg/endpoints/components_install_uninstall.go

```diff
@@ -0,0 +1,130 @@
+package endpoints
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/airyhq/airy/lib/go/payloads"
+	helmCli "github.com/mittwald/go-helm-client"
+	"helm.sh/helm/v3/pkg/repo"
+	"k8s.io/client-go/rest"
+	"k8s.io/klog"
+)
+
+type ComponentsInstallUninstall struct {
+	cli       helmCli.Client
+	namespace string
+}
+
+func MustNewComponentsInstallUninstall(namespace string, kubeConfig *rest.Config) ComponentsInstallUninstall {
+	cli, err := helmCli.NewClientFromRestConf(&helmCli.RestConfClientOptions{
+		Options: &helmCli.Options{
+			Namespace: namespace,
+		},
+		RestConfig: kubeConfig,
+	})
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	//NOTE: For now we are setting the repos here. But in the future we could get them from an external configuration
+	chartRepoCore := repo.Entry{
+		Name: "airy-core",
+		URL:  "https://helm.airy.co",
+	}
+	if err := cli.AddOrUpdateChartRepo(chartRepoCore); err != nil {
+		log.Fatal(err)
+	}
+
+	//NOTE: We Look for user and password for airy-enterprise. If found we add it to our helm client.
```

Comment:
nitpick: We shouldn't add code for things that we haven't decided yet on. Until we do this code is dead weight.

Reply:
Yes I moved this to a config-map